draft-ppm-protocol.md

```diff
@@ -409,22 +419,31 @@ struct {
 
 ## Task Configuration {#task-configuration}
 
-Prior to the start of execution of the protocol, each participant must agree on
-the configuration for each task. A task is uniquely identified by its task ID:
+A task centers around evaluation of a specific Verifiable Distributed
+Aggregation Function (VDAF, see [VDAF]) over a set of client inputs. The VDAF
+determines the type of input measurements and the type of the output of the
+aggregation function. It also determines how many aggregators are involved (this
+document currently supports just two) and how many rounds of communication among
+the aggregators are required to verify an input's validity. Prior to the start
+of execution of the protocol, each participant must agree on the configuration
+parameters for the task. A task is uniquely identified by its task ID:
 
 ~~~
 opaque TaskId[32];
 ~~~
 
-A `TaskId` is a globally unique sequence of bytes. It is RECOMMENDED that this
-be set to a random string output by a cryptographically secure pseudorandom
-number generator. Each task has the following parameters associated with it:
+A `TaskId` is a unique identifier for a specific task among a set of tasks. It
+is RECOMMENDED that this be set to a random string output by a cryptographically
+secure pseudorandom number generator. Each task has the following parameters
+associated with it:
 
 * `aggregator_endpoints`: A list of URLs relative to which an aggregator's API
   endpoints can be found. Each endpoint's list MUST be in the same order. The
   leader's endpoint MUST be the first in the list. The order of the
   `encrypted_input_shares` in a `Report` (see {{uploading-reports}}) MUST be the
-  same as the order in which aggregators appear in this list.
+  same as the order in which aggregators appear in this list. The length of this
+  list MUST be two. [NOTE We intend to support multiple helpers in the future,
```

Comment:
This restriction should appear in precisely one place so that we do not have to change it everywhere if we add multiple helpers. The first location is probably the right one.

Reply:
Good call. I removed this MUST here and the paranthetical above saying the protocol is only defined for two aggregators. I added a MUST for this at the end of {{task-configuration}}.